packages/holocron/__tests__/HolocronErrorBoundary.spec.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
why is this needed?

Reply:
This is needed because `enzyme` is at the parent package.json level and not the `holocron` package.json level. It is done the same way here https://github.com/americanexpress/holocron/blob/master/packages/holocron/__tests__/RenderModule.spec.jsx#L17-L18